### ISSUE TITLE
Info about Lobster?

### ISSUE THREAD

**Issue by tedstoychev** [vlang/v#17100]
### Describe the issue

V's memory management model is based on the one from the Lobster programming language:

https://aardappel.github.io/lobster/memory_management.html

However this info isn't mentioned anywhere on V's homepage or documentation. As a result, this question gets raised repeatedly on Discord and other platforms.

I would like to suggest mentioning Lobster's memory management model on V's website/docs. A sentence like "V's memory management model is inspired by the Lobster programming language." should suffice.

Potential places where this info could be added:

\- 'Acknowledgement' section in the Readme:
https://github.com/vlang/v#acknowledgement

\- 'Memory management' section in the docs:
https://github.com/vlang/v/blob/master/doc/docs.md#memory-management

\- 'Innovative and flexible memory management' section on V's homepage:
https://vlang.io/#memory

### Links

https://github.com/vlang/v#acknowledgement
https://github.com/vlang/v/blob/master/doc/docs.md#memory-management
https://vlang.io/#memory

**Comment by i582** [comment]
Hey

> V's memory management model is based on the one from the Lobster programming language:

Why do you think so? I myself am interested in how the memory management model works, so if you looked at V's source code and understood it, then tell me.

**Comment by i582** [comment]
@JalonSolov memory management is probably one of the most important components of the language. When you write "innovative memory management system" you are just marketing, and when there is no description AT ALL, it becomes like a something bad. 

Lack of documentation just makes the language less obvious to users. Moreover, increases the number of so-called trolls.

Even just mentioning that the system is based on Lobster removes a lot of questions.

**Comment by JalonSolov** [comment]
I agree that memory management is an important aspect of a language.  I don't see documenting it as more important than making it all work properly.

Yes, it does need to be properly documented at some point, including similarities/differences with other systems, etc.

**Comment by tedstoychev** [comment]
To clarify, I was referring to autofree. I was left with this impression from:

\- discussions on Discord (you can search for "lobster" and sort by oldest comments first)
\- comments on Reddit by /u/volt_dev (I assume this is Alex?)

**Comment by medvednikov** [comment]
It's not based on Lobster. I designed it before I came across Lobster. It's similar to Lobster and Nim. That's it.

It uses very simple compile time management principles. It's described in detail on the home page:

> Autofree can be enabled with -autofree. It takes care of most objects (~90-100%): the compiler inserts necessary free calls automatically during compilation. Remaining small percentage of objects is freed via GC. 

That's all there's to it.
